### DISCUSSION TITLE
The Information Theory, Pattern Recognition, and Neural Networks - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#439] [Announcements]
# The Information Theory, Pattern Recognition, and Neural Networks - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%95%B0%E5%AD%A6%E8%BF%9B%E9%98%B6/The_Information_Theory_Pattern_Recognition_and_Neural_Networks/

**Comment by MTDickens** [comment]
补充 YouTube 视频列表（有字幕）：https://www.youtube.com/playlist?list=PLruBu5BI5n4aFpG32iMbdWoRVAA-Vcso6

**Comment by MTDickens** [comment]
另外 mackay 讲得很广很杂，建议第一遍就先顺着看一遍，之后再慢慢回味。另外 mackay 的课上其实没有讲很多东西，如果希望仔细了解的话，还是要看书。

**Comment by melodygui** [comment]
大家有statistical machine learning相关的课程推荐么？

**Comment by Goer17** [comment]
R.I.P

**Comment by lil-Journey** [comment]
自建日本节点服务器 月租5元不限流量 本人也在用 失效时可联系人工修复 qq:1165116287

**Comment by tyLingyu** [comment]
R.I.P
